iceoryx_binding_c/test/moduletests/test_subscriber.cpp

```diff
@@ -245,3 +259,52 @@ TEST_F(iox_sub_test, sendingTooMuchLeadsToLostChunks)
 
     EXPECT_TRUE(iox_sub_has_lost_chunks(m_sut));
 }
+
+TEST_F(iox_sub_test, attachingToWaitSetWorks)
+{
+    EXPECT_EQ(iox_sub_attach_to_ws(m_sut, m_waitSet, SubscriberEvent_HAS_NEW_SAMPLES, 0, NULL), WaitSetResult_SUCCESS);
+    EXPECT_EQ(m_waitSet->size(), 1);
+}
+
```

Comment:
Attaching multiple times? Detaching not attached subscriber?

Reply:
I missed this and this will be added.

Reply:
It is added.